go.mod

```diff
@@ -48,7 +48,10 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-require google.golang.org/protobuf v1.27.1
+require (
+	github.com/onsi/ginkgo v1.16.4
```

Comment:
github.com/onsi/ginkgo/v2 is already added in the above require. No need to add a second (older) instance of ginkgo.